.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
 dist: xenial
+virt: lxd
```

Comment:
Just reading about this now. Neat. Do you want to add `arch: arm64-graviton2` too?

Reply:
Would that speed up the arm builds?

Reply:
Allegedly. https://blog.travis-ci.com/2020-09-11-arm-on-aws
https://blog.travis-ci.com/AWS-Graviton-2-support-comes-to-Travis-CI

Reply:
I've tried it and it's broken: https://travis-ci.org/github/ionelmc/python-hunter/jobs/740593212

It looks like it actually runs amd64 (probably due to a travis bug).